### DISCUSSION TITLE
Keybindings are inverted on 1 machine, not the other

### DISCUSSION THREAD

**Discussion by morris14** [LazyVim/LazyVim#398] [General]
Having an odd issue where my keybindings are reversed on one of my machines, but fine on the other, even though they both use the exact same config from my dotfiles repo.

On 1 machine, if i press `<Leader>ff`, the search will open to search from the root of the project and `<Leader>fF` will search from the CWD. Whereas on my other machine, the keybindings are flipped...no idea why.

I'm using Kitty, and the config for Kitty references the same file on both machines.

**Comment by folke** [comment]
Do you have caps lock enabled on one of the machines by any chance?

**Reply by morris14** [reply]
Afraid not!

**Reply by folke** [reply]
Then no idea. Can't help you. I'm pretty sure this is not related to lazyvim

**Reply by morris14** [reply]
No worries, thanks @folke
